src/lib/shared/User.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+	import { ethers } from 'ethers';
+	const provider = ethers.getDefaultProvider();
```

Comment:
Going to remove this in my next pr as we probably want to use an injected provider instead of the ethers default one.